packages/popover/src/Popover.story.tsx

```diff
@@ -40,15 +76,24 @@ export default {
     children: {
       control: false,
     },
+    refButtonPosition: {
+      options: ['centered', 'top', 'right', 'bottom', 'left'],
+      control: { type: 'select' },
+    },
   },
 } as Meta<typeof Popover>;
 
-type PopoverStoryProps = PopoverProps & { buttonText: string };
+type PopoverStoryProps = PopoverProps & { buttonText: string } & {
+  refButtonPosition: string;
+};
```

Comment:
I think we can just use one type here instead of two?
```
type PopoverStoryProps = PopoverProps & { 
  buttonText: string
  refButtonPosition: string;
};
```

Reply:
Not sure if this works, but it would be nice to indicate using TSDocs that these props are only for Storybook. Storybook doesn't differentiate these props from the component's props, so it might be confusing. I've been planning to do this myself on other components.

Reply:
i think i figured it out